google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -131,7 +131,7 @@ class Block:
             Required. UTF-8 encoded text.
     """
 
-    documentai_block: documentai.Document.Page.Block
+    documentai_block: dataclasses.InitVar[documentai.Document.Page.Block]
```

Comment:
Another small inconsistency is that now only some of the wrapper classes require the `shard_index` init argument, but not all.  Can we make it easy for the next engineer to tell how to decide whether a wrapper class should vs should not have `shard_index`?

Reply:
Removed these